### DISCUSSION TITLE
This app can't run on your PC

### DISCUSSION THREAD

**Discussion by TsieOmni** [zhongfly/mpv-build#116] [Q&A] [answered]
I am running win10 OS and and getting the below error when i try to install, what could be the reason for this? I've tried everything, ran it as admin, fan capabilities troubleshooter, ran dev mode and still it refuses. :'( 
This app can't run on your PC
*To find a version for your PC, check with the software publisher*

**Comment by zhongfly** [comment]
If you are using 64 bit system:
1. download `mpv-x86_64-xxxx`
2. If your cpu is newer than 2015, you can also try `mpv-x86_64-v3-xxx`

If you are using 32 bit system,please download `mpv-i686-xxx` from
 https://github.com/shinchiro/mpv-winbuild-cmake

If you are using Windows 10 ARM64, download  `mpv-aarch64-xxxx`

**Reply by TsieOmni** [reply]
Thank you let me try em out, i revert back

**Reply by TsieOmni** [reply]
![Screenshot (51)](https://github.com/user-attachments/assets/51d4a1a1-31a5-415b-91d1-6a224efb9136)
idk maybe i am trippin, but the ONLY folder with an installation file is the *mpv-aarch64-20241103-git-42ff6f9* ,
i am running a 64bit win10 OS, and my CPU is older than 2015...what to do now??

**Reply by zhongfly** [reply]
Show all 21 assets:
![Screenshot_20241108182102](https://github.com/user-attachments/assets/3ef75429-7e0d-4490-af17-9c0629f0a403)

**Reply by zhongfly** [reply, marked as answer]
https://github.com/zhongfly/mpv-winbuild/releases/download/2024-11-08-3bbc770/mpv-x86_64-20241108-git-3bbc770.7z

**Reply by TsieOmni** [reply]
![Screenshot (52)](https://github.com/user-attachments/assets/57ae39b6-6ff6-4605-9b40-1a02c00748de)
so here is my dilemma, so my application on VS code indicates that i need the mpv to stream/playaudio when i run it.
when i try to install the .exe files you suggested an audio player pops up instead...unless i am enquiring the incorrect application...i am confused.

**Reply by zhongfly** [reply]
https://github.com/elevenlabs/elevenlabs-python/blob/243f851d7a9af79dbb615a6ccd8bd053db07ae66/src/elevenlabs/play.py#L70
`is_installed("mpv")`
https://github.com/elevenlabs/elevenlabs-python/blob/243f851d7a9af79dbb615a6ccd8bd053db07ae66/src/elevenlabs/play.py#L6
so `shutil.which("mpv")` not found mpv
https://docs.python.org/3/library/shutil.html#shutil.which

So problem is: `mpv.exe` and `mpv.com` not in `PATH` environment variable.

You need to look deeper to find the source of the problem.
